plugins/woocommerce/client/admin/client/settings-payments/index.tsx

```diff
@@ -194,7 +197,21 @@ const SettingsPaymentsMethods = () => {
 	// Retrieve wooPayments gateway
 	const wooPayments = getWooPaymentsFromProviders( providers );
 
-	const onClick = useCallback( () => {
+	const onPaymentMethodsContinueClick = useCallback( () => {
+		// Record the event along with payment methods selected
+		recordEvent( 'wcpay_settings_payment_methods_continue', {
```

Comment:
I think we should only pass string values for event props, not objects. Here is [an example](https://github.com/woocommerce/woocommerce/blob/5d1959c5f3305e1a702d4a12f97ad04dd825354a/plugins/woocommerce/client/admin/client/payments-welcome/index.tsx#L119-L119) of turning a list into a coma-separated string for an event props.

AFAIK, Tracks only handles single values event props. Even booleans get translated. See this Slack message p1714122766970679-slack-C03KTTK2YMA.

We are best to rely on some Tracks ingestion system (that may or may not be there) to automatically convert things.

Reply:
cc @dmallory42

Reply:
Great catch, @vladolaru!

Reply:
Great catch! I missed this nuance when looking at the usage of recordEvent in our codebase. Fixed in https://github.com/woocommerce/woocommerce/pull/54733/commits/d53a9fa0de9913dd9bdd8875a6550ea45b1ebfed.